redis_rate_limit/__init__.py

```diff
@@ -88,7 +90,6 @@ def get_usage(self):
         """
         Returns actual resource usage by client. Note that it could be greater
         than the maximum number of requests set.
-
```

Comment:
You've removed a bunch of blank lines from docstrings. Could you revert it, please?

Reply:
changes reverted.